tsweb/debug.go

```diff
@@ -37,6 +37,8 @@ type DebugHandler struct {
 	title    string                           // title displayed on index page
 }
 
+var PrometheusHandler feature.Hook[func(*DebugHandler)]
```

Comment:
We should document this, at least to the point of how to get it set.
```suggestion
// PrometheusHandler is an optional hook to enable native Prometheus support
// in the debug handler. It is disabled by default. Import the "promvarz"
// package to enable this feature.
var PrometheusHandler feature.Hook[func(*DebugHandler)]
```

or words to that broad effect.

Reply:
Done.